loadsmart/datahub
# feat: [ROCK-309] change welcome modal from DataHub to Data Catalog

# ROCK-309: Change welcome modal message from DataHub to Data Catalog

## Description

Add Data Catalog's custom text message and image on the welcome modal that appears for new users.

## Checklist

This PR complies to:

- [ ] (_N/A_) ~Documented (API SPECs, software architecture changes, and code comments where necessary)~
- [x] Smallest size possible
- [x] Synced branch with master
- [ ] Tested on QA
- [ ] (_N/A_) ~Clog entry (if needed)~

## Requirements

_N/A_

## How to test

1. Start the frontend
```shell
cd datahub-web-react
yarn
yarn start:mock
```
2. Login with an [available test user](src/graphql-mock/fixtures/searchResult/userSearchResult.ts#L6)
3. Check the results

## Showroom

_TBA_

## Metrics

_N/A_
